server/collaborator/src/platform.ts

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright © 2023 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import client from '@hcengineering/client'
+import clientResources from '@hcengineering/client-resources'
+import core, { Client, TxOperations } from '@hcengineering/core'
+import { setMetadata } from '@hcengineering/platform'
+import { Token } from '@hcengineering/server-token'
+import config from './config'
+
+// eslint-disable-next-line
+const WebSocket = require('ws')
+
+export async function connect (transactorUrl: string, token: string): Promise<Client> {
```

Comment:
We definitely need this method to be part of our server API, we had it 5-6 times already:-)

Reply:
Opened https://front.hc.engineering/workbench/platform/tracker/UBERF-4734